### PR TITLE
feat: added cli option titlePage

### PR DESCRIPTION
**Pull request by johnthad**
This adds the option `--titlePage` to allow a user to configure the text to something of their choice. No using the option keeps the default text 'Titlepage`.

### CODE REVIEW

**Comment by wshito** [comment]
@johnthad Thanks for the contribution.  I took a look at the patch and it looks good.  More choice in toc is certainly better.  I will apply the patch and see how it works by this weekend.  By the way, is it possible to add the descriptions in README too?

**Comment by johnthad** [comment]
> @johnthad Thanks for the contribution. I took a look at the patch and it looks good. More choice in toc is certainly better. I will apply the patch and see how it works by this weekend.

Thank you. It looks like I'm moving from `DocBook` to `Asciidoctor` for my current project, and this chunker fills a void. If you like I can add a section to `README.md` like `## Customizing Tltlepage` later today.

File: src/CommandOptions.mjs [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const makeConfig = (argv) => {
     .option('--depth <depth specifier>', 'See the description above.', '1')
     .option('--css <paths>', 'The comma seprated list of css file paths to include.  The paths must be accesible from the current directory.', commaSeparatedList)
     .option('--no-strictMode', 'Turns off the strict mode.')
+    .option('--titlePage <title string>', 'Sets title page string.', 'Titlepage')
```
Comment (wshito) [inline src/CommandOptions.mjs]: Maybe 'Sets title page **_toc_** string' is clearer?  English is not my first language so I am not sure if it is correct though.

Comment (johnthad) [reply src/CommandOptions.mjs]: The toc title is set by `:toc-title:` attribute in the `*.doc`. I was specifically getting at the word `Titlepage` appearing just underneath the toc title. See attached screenshot.
Screen Shot 2021-04-27 at 11 01 15

Comment (wshito) [reply src/CommandOptions.mjs]: Yes, I know what you mean as I read the your patch.  But when I first read the descriptions in `CommandOptions.mjs`, I thought you implemented the way to customize the main title or some kind of title page contents from the command line.  So I checked the patch and found out that you wanted to customize the toc item for the  top page link.   I don't know any better ways to describe in English than as I suggested in the previous comment.

Comment (johnthad) [reply src/CommandOptions.mjs]: Okay. Yes, your wording could help clarify its use. I can't think of an alternative.

Comment (johnthad) [reply src/CommandOptions.mjs]: Done.

**Comment by wshito** [comment]
>  If you like I can add a section to `README.md` like `## Customizing Tltlepage` later today.

Yes, that'll help me a lot!

**Comment by wshito** [comment]
@johnthad Thank you very much for updating README.  I will test the new option and merge your request soon.

**Comment by wshito** [comment]
@johnthad  Thank you for your contribution!

**Comment by johnthad** [comment]
You're welcome. Will a new version be on npm soon?

**Comment by wshito** [comment]
Yes, I will put the tag for the new version and release it on npm, hopefully this weekend!

**Pull request merged**